package.json

```diff
@@ -25,6 +25,10 @@
   "keywords": [],
   "author": "typicode <typicode@gmail.com>",
   "license": "SEE LICENSE IN ./LICENSE",
+  "repository": {
+		"type": "git",
+		"url": "git+https://github.com/typicode/json-server.git"
+	},
```

Comment:
argh. fcking code style again.

i made edit from web, copied from typicode package.json and fixed path.

i guess here can be no git hooks to prevent it, it should be formatted on CI pipeline and aborted if mismatch.

you didn't not notice this yourself when looking diffs in github files view?
- https://github.com/typicode/json-server/pull/1511/files

Reply:
I'm doing it on my phone on the GitHub app lol and I still don't see the issue. It's not published so it's fine. Thanks for the heads up.

Reply:
Oh, the tabs I guess?

Reply:
yes tabs.

<img width="422" alt="image" src="https://github.com/typicode/json-server/assets/199095/6fe9dd8e-7190-4270-81b9-3b48e323faaf">

Reply:
I think any command like `npm version` `npm install` should fix it (which happen fairly often with v1 work). You can make a PR or leave it as it is, as you prefer :)

Reply:
I'll let you to finish it :)

however,
I just randomly read husky install code. and you may not like what it does. the way I read it out is if it finds single tab in the file it will format whole file with tabs.

Reply:
UPDATE:

but, the code is buggy as well, so it doesn't work?
- https://github.com/typicode/husky/commit/ec13855c167bcead06523ae43ca1b5152d77810f#r138138417